framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -262,7 +315,8 @@ public void run() {
   }
   
   public synchronized void setTestStatus(TestStatus status) {
-	if (testStatus == TestStatus.CANCELED) return;
+	if (testStatus == TestStatus.CANCELED || testStatus == TestStatus.VERIFICATION_FAILURE) 
```

Comment:
There isn't anything that does that, right? Data verification happens beforehand, so it seems unnecessary. 

Also in execLimitZeroQuery, there should be a throw new VerificationException(...) rather than set status and exception directly. The run method takes care of failing the test. 

> On Dec 2, 2015, at 9:50 PM, Chun Chang <notifications@github.com> wrote:
> 
> In framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java:
> 
> > @@ -262,7 +315,8 @@ public void run() {
> >    }
> >    
> >    public synchronized void setTestStatus(TestStatus status) {
> > -	if (testStatus == TestStatus.CANCELED) return;
> > +	if (testStatus == TestStatus.CANCELED || testStatus == TestStatus.VERIFICATION_FAILURE) 
> Yes, necessary. We may have other verification try to set it to success. It's a good defensive coding.
> 
> —
> Reply to this email directly or view it on GitHub.
>